src/build/app/mod.rs

```diff
@@ -1952,8 +1952,7 @@ impl<'help> App<'help> {
         Ok(matcher.into_inner())
     }
 
-    // used in clap_generate (https://github.com/clap-rs/clap_generate)
-    #[doc(hidden)]
+    /// Some properties of an `App`
```

Comment:
Should this not be hidden?

Reply:
@CreepySkeleton I think you missed this.

Reply:
Um, yes, I was going to address the thing in this PR but then figured it would be too off-topic. This chunk somehow slipped through, oops.